lib/liquid/block_body.rb

```diff
@@ -70,6 +70,21 @@ def render(context)
       render_to_output_buffer(context, '')
     end
 
+    def format(output)
+      idx = 0
+      while node = @nodelist[idx]
+        output << case node
```

Comment:
Why not inline this entire expression?
```ruby
    def format(output)
      idx = 0
      while node = @nodelist[idx]
        output << node.is_a?(String) ? node : node.format
        idx += 1
      end

      output
    end
```

Reply:
I'll like do this sort of tweak in the final version. This is still likely to change and was originally more case statements. If it ends up being just these two then yes this makes sense.